pkg/preflights/preflights.go

```diff
@@ -61,6 +63,28 @@ func Run(ctx context.Context, spec *troubleshootv1beta2.HostPreflightSpec, proxy
 	return out, stderr.String(), err
 }
 
+func CopyBundleToECLogsDir() error {
+	matches, err := filepath.Glob("preflightbundle-*.tar.gz")
+	if err != nil {
+		return fmt.Errorf("unable to find preflight bundle: %w", err)
+	}
+	if len(matches) == 0 {
+		return nil
+	}
+	// get the newest bundle
+	src := matches[0]
+	for _, match := range matches {
+		if filepath.Base(match) > filepath.Base(src) {
+			src = match
+		}
+	}
+	dst := filepath.Join(defaults.EmbeddedClusterLogsSubDir(), "preflight-bundle.tar.gz")
```

Comment:
this shouldn't be unique - I think for logs we use the timestamp or something?

That way we don't overwrite each time the install command is run

Reply:
My thinking here is that if we store all these bundles here it will bloat the support bundle too much so i am only storing the latest.